#--- a/pytest.ini
#+++ b/pytest.ini
#@@ -1,3 +1,3 @@
# [pytest]
# testpaths = tests
#-addopts   = -v --cov=fsleyes_widgets --cov-report=html --html=report.html
#+#addopts   = -v --cov=fsleyes_widgets --cov-report=html --html=report.html
--- a/setup.py
+++ b/setup.py
@@ -107,13 +107,6 @@
 
     packages=packages,
     install_requires=install_requires,
-    setup_requires=['pytest-runner', 'sphinx', 'sphinx-rtd-theme', 'mock'],
-    tests_require=['mock',
-                   'coverage',
-                   'pytest-cov',
-                   'pytest-html',
-                   'pytest-runner',
-                   'pytest'],
     test_suite='tests',
 
     cmdclass={
